benchmark/webstreams/pipe-to.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+const common = require('../common.js');
+const {
+  ReadableStream,
+  WritableStream,
+} = require('node:stream/web');
+
+const bench = common.createBenchmark(main, {
+  n: [5e6],
+});
+
+
+async function main({ n }) {
+  const b = Buffer.alloc(1024);
+  let i = 0;
+  const rs = new ReadableStream({
+    pull: function(controller) {
```

Comment:
Can you include `strategy` to the benchmarks?

Reply:
I'll include the `strategy.size`. For `highWaterMark` would be better in a separate file. To handle backpressure and so on.

Reply:
Actually, both don't make sense to include at this benchmark, they won't diverge. Anyway, I've included highWaterMark value for both ReadableStream and WritableStream.